app.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+from app.converter import Converter
```

Comment:
```suggestion
from app import Converter
```

I'd hoist Converter inside the app's `__init__`